src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { IconButton, IconButtonSize } from '../../enums/icon-button'
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms'
+
+export interface MultiSelectOption<T> {
+  name: string
+  value: T
+}
+
+interface SelectableOption<T> extends MultiSelectOption<T> {
+  isSelected: boolean
+}
```

Comment:
The idea behind `SelectableOption` was so that `MultiSelectComponent` could internal know if the option is selected without having all callers of the MultiSelect component having to write `isSelected: false` for all their options and without making `isSelected` optional.